docs/src/proxy.md

```diff
@@ -41,6 +41,11 @@ A session represents ongoing communication flow between a client and an [Upstrea
 
 The proxy exposes the following general metrics (See the metrics sub-sections for metrics specific to other Quilkin components, e.g for metrics related to packet flow see [sessions metrics][session-metrics], or metrics exported by individual filters can be found in the documentation for each filter):
 
+- `quilkin_proxy_processing_time_seconds{event}` (Histogram)
```

Comment:
```suggestion
- `quilkin_proxy_packet_processing_duration_seconds{event}` (Histogram)
```
we can clarify that it only concerns packets, then `duration` is more conventional for prometheus histograms